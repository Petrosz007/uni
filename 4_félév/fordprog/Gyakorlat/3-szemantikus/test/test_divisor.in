2
5
121
35
10000
100001
0
