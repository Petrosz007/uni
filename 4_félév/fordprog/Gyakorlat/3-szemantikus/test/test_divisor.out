false
false
true
11
true
5
true
2
true
11
